server/handler.go

```diff
@@ -45,20 +45,32 @@ func (p *MatterpollPlugin) handleVote(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	displayName, appErr := p.ConvertUserToDisplayName(poll.Creator)
+	if appErr != nil {
+		response.EphemeralText = commandGenericError
+		writePostActionIntegrationResponse(w, response)
+		return
+	}
+
 	hasVoted := poll.HasVoted(userID)
 	err := poll.UpdateVote(userID, optionNumber)
 	if err != nil {
 		response.EphemeralText = commandGenericError
 		writePostActionIntegrationResponse(w, response)
 		return
 	}
+
 	appErr = p.API.KVSet(pollID, poll.Encode())
 	if appErr != nil {
 		response.EphemeralText = commandGenericError
 		writePostActionIntegrationResponse(w, response)
 		return
 	}
 
+	post := model.Post{}
```

Comment:
If we can use `PostActionIntegrationResponse.PostId`, we can get Post instance, should change only `Post.Text`, and can avoid to add `SiteURL` to `MatterpollPlugin`.
However, even though this PR was merged, `PostActionIntegrationResponse.PostId` looks having no value now.

Do you know why can't get `PostActionIntegrationResponse.PostId`?

Reply:
I'm on the latest master branch and developer environment.
"Delete Poll" still reply `This feature is only available on Mattermost v5.3.`.

Reply:
Lets continue the discussion on pre-release